.github/workflows/render-rmarkdown.yaml

```diff
@@ -0,0 +1,68 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+name: render-rmarkdown.yaml
+
+permissions: read-all
+
+jobs:
+  render-rmarkdown:
+    runs-on: ubuntu-latest
+    # Only restrict concurrency for non-PR jobs
+    concurrency:
+      group: rmarkdown-${{ github.event_name != 'pull_request' || github.run_id }}
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+      
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 'latest'
+
+      - name: Setup Pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Setup R Dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::rmarkdown
+
+      - name: Render RMarkdown files
+        shell: bash
+        run: |
+          RMD_PATH=$(git diff --name-only ${{ github.event.before }} ${{ github.sha }} | grep '\.Rmd$' || true)
```

Comment:
This is awesome, but just a note here:
``` bash
RMD_PATH=$(git diff --name-only ${{ github.event.before }} ${{ github.sha }} | grep '\.Rmd$' || true)
```

This bash command will run a `git diff` and `grep` any `.Rmd` files that have been changed before/ after the PR. While that will catch any direct changes to the `.Rmd` itself, it _won't_ catch underlying changes in the `blazr` package. 

So for example, if we were to update the functionality of a `blazr` function but didn't make any changes to the `README`, this action wouldn't catch those changes, and wouldn't try to trigger a re-knit of the `README.Rmd`. 

Certainly not a blocking comment to this PR, but maybe something worth expanding on at a later date to improve the action

Reply:
Suggest to extract this comment into a new tracking issue

Reply:
Created new issue #18